bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -432,65 +435,66 @@ frame_support::construct_runtime! {
 
 		// Consensus support.
 		Authorship: pallet_authorship::{Module, Call, Storage, Inherent} = 7,
-		Staking: darwinia_staking::{Module, Call, Storage, Config<T>, Event<T>, ValidateUnsigned} = 8,
-		Offences: pallet_offences::{Module, Call, Storage, Event} = 9,
-		Historical: pallet_session_historical::{Module} = 10,
-		Session: pallet_session::{Module, Call, Storage, Config<T>, Event} = 11,
-		Grandpa: pallet_grandpa::{Module, Call, Storage, Config, Event, ValidateUnsigned} = 12,
-		ImOnline: pallet_im_online::{Module, Call, Storage, Config<T>, Event<T>, ValidateUnsigned} = 13,
-		AuthorityDiscovery: pallet_authority_discovery::{Module, Call, Config} = 14,
+		ElectionProviderMultiPhase: pallet_election_provider_multi_phase::{Module, Call, Storage, Event<T>, ValidateUnsigned} = 8,
```

Comment:
The pallet ids are changed, will it have side effect or impacts unexpected?

It might be ok for Pangolin since it will be reset, but do we need to care the convention of compatibility on production networks (Crab and Darwinia).

Reply:
In Crab/Darwinia, we just need to increase the pallet index by 1.